datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedTaskCountEstimator.java

```diff
@@ -5,18 +5,32 @@
  */
 package com.linkedin.datastream.server.assignment;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
-import org.apache.commons.lang.NotImplementedException;
+import org.apache.commons.lang3.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.annotations.VisibleForTesting;
 
 import com.linkedin.datastream.server.ClusterThroughputInfo;
+import com.linkedin.datastream.server.PartitionThroughputInfo;
 
 
 /**
  * Estimates the minimum number of tasks for a datastream based on per-partition throughput information
  */
 public class LoadBasedTaskCountEstimator {
 
+  private static final Logger LOG = LoggerFactory.getLogger(LoadBasedTaskCountEstimator.class.getName());
+
+  // TODO Make this constant configurable
+  @VisibleForTesting
+  public final static int TASK_CAPACITY_MBPS_DEFAULT = 4;
```

Comment:
since it is visible for testing should you remove the `final` ?

Reply:
I'd prefer to leave it as it is.